src/output.cpp

```diff
@@ -294,7 +294,7 @@ std::string Output::value_to_str(BPFtrace &bpftrace,
     return res.str();
   }
   else
-    return std::to_string(read_data<int64_t>(value.data()) / div);
+    return "";
```

Comment:
I think that this was originally intended to cover situations when we forget to add a handle for certain type. Maybe it makes sense to preserve that and return `""` only for `Type::None` and/or when `value` is empty?

Reply:
The problem is that unsupported types won't necessarily always be 8-bytes. If they're smaller then the always-printing-something approach has the potential to be undefined behaviour by reading uninitialised memory. If they're larger then we'll be silently truncating values and people might not notice immediately.

IMO an obvious failure is better than a subtle one and crashes should be avoided at all costs.

The best solution would be for this function to use an exhaustive switch-statement instead of a chain of ifs, but I'll save that refactoring for another time.

Reply:
Ok, makes sense.